packages/paste-website/src/pages/components/chat-composer/index.mdx

```diff
@@ -202,15 +286,15 @@ export const BasicChatComposer = () => (
-| `placeholder?`  | `string` or `JSX.Element`                                   | Placeholder content for the Chat Composer                                                 |                   |
-| `onChange?`     | `(editorState: EditorState, editor: LexicalEditor) => void` | Callback invoked when changes occurs within the Chat Composer                             |                   |
-| `initialValue?` | `string`                                                    | Initial value for the Chat Composer                                                       |                   |
-| `maxHeight?`    | `string`                                                    | Maximum height for the Chat Composer                                                      |                   |
+| Prop            | Type                                                                                       | Description                                                                               | Default           |
+| --------------- | ------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------- | ----------------- |
+| `config`        | `LexicalComposerProps['initialConfig']`                                                    | Configuration provided to the Lexical Composer                                            |                   |
+| `children?`     | `LexicalComposerProps['children]`                                                          | Children provided to the Lexical Composer (i.e. Lexical plugins)                          |                   |
+| `element?`      | `string`                                                                                   | Overrides the default element name to apply unique styles with the Customization Provider | `'CHAT_COMPOSER'` |
+| `placeholder?`  | `string` or `JSX.Element`                                                                  | Placeholder content for the Chat Composer                                                 |                   |
+| `onChange?`     | `onChange?: (editorState: EditorState, editor: LexicalEditor, tags: Set<string>) => void;` | Callback invoked when changes occurs within the Chat Composer                             |                   |
+| `initialValue?` | `string`                                                                                   | Initial value for the Chat Composer                                                       |                   |
+| `maxHeight?`    | `string`                                                                                   | Maximum height for the Chat Composer                                                      |                   |
 
 ### Figma
 
```

Comment:
suggestion: remove the Figma callout here, and replace it with a paragraph:

You can find the Chat Composer components in [the Conversations UI Kit section of the Paste Components library.](https://www.figma.com/file/l2qtpQVMmXzQPVB3rEAA8i/Paste-Components?type=design&node-id=15140-99225&mode=design&t=6zHUtRXfN2GWTR0g-0)